control-plane/api/common/consul_resource_webhook_test.go

```diff
@@ -55,7 +55,7 @@ func TestValidateMeshConfig(t *testing.T) {
 			expErrMessage: "invalid",
 		},
 		"duplicate name": {
-			existingResources: []MeshConfig{&mockMeshConfig{
+			existingResources: []ConsulResource{&mockMeshConfig{
```

Comment:
in the spirit of completeness, would you mind updating the name of the mock as well?